src/libmagickwand.jl

```diff
@@ -20,27 +20,29 @@ export MagickWand,
 
 # Find the library
 depsfile = joinpath(dirname(@__FILE__), "..", "deps", "deps.jl")
-versionfile = joinpath(dirname(@__FILE__), "..", "deps", "versioninfo.jl")
 
+init_envs = Dict{String,String}()
 if isfile(depsfile)
     include(depsfile)
 else
     error("ImageMagick not properly installed. Please run Pkg.build(\"ImageMagick\") then restart Julia.") # now that this is decoupled from images, should this be an error?
 end
-if isfile(versionfile)
-    include(versionfile)
-end
 
 const have_imagemagick = isdefined(:libwand)
 
 # Initialize the library
 function __init__()
-    init_deps()
+    for (key, value) in init_envs
+        ENV[key] = value
+    end
     !have_imagemagick && warn("ImageMagick utilities not found. Install for more file format support.")
+    ccall((:MagickWandGenesis, libwand), Void, ())
+    p = ccall((:MagickQueryConfigureOption, libwand), Ptr{UInt8}, (Ptr{UInt8}, ),
+              "LIB_VERSION_NUMBER")
+    global const libversion = VersionNumber(join(split(unsafe_string(p), ',')[1:3], '.'))
```

Comment:
https://github.com/JuliaLang/julia/issues/12010 indeed suggests that it's better not to use this. We could define it at compile-time, but I don't think it'll be clear to users that `libversion` might not reflect the actual version of the library.

We could also follow PyCall's solution: https://github.com/JuliaPy/PyCall.jl/blob/1d755f27fd440a43b9a792919fee0531495754e0/src/pyinit.jl#L50, where the version number seems to be defined at compile time, but checked at runtime.

Another alternative is to declare `libversion = Ref{VersionNumber}()` and update its value during `__init__`. This would mean that users have to call `ImageMagick.libversion[]` to get the version number.

Or maybe you have some other alternative?